.circleci/config.yml

```
@@ -660,7 +651,7 @@ jobs:
       - attach_workspace: {at: /tmp}
       - *vendor_bin_add_path
       - run: "drush pm:security"
-      - run: "drush pm:security-php"
+      - run: "composer audit"
```

```suggestion
      - run: "drush pm:security-php"
```
I recommend rolling this change back to get other fixes in this release. Will merge in and update the ticket